packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -208,6 +212,18 @@ export const filterLocale = (
     localeItems.length === localeItemsHidden.length
       ? modalLabels.unavailabilityText
       : modalLabels.availabilityText;
+
+  /**
+   *  Reflect number of results in the accessibility readout
+   */
+  const resultsElement = document.querySelector(`.${prefix}--assistive-text`);
```

Comment:
This `querySelector()` call has to be closely scoped. Note that `bx--assistive-text` is general-purpose CSS class for "hidden visually but visible from screen reader" elements.